src/ripple/server/impl/Port.cpp

```diff
@@ -205,6 +205,29 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
         }
     }
 
+    {
+        auto const result = section.find("send_queue_limit");
+        if (result.second)
+        {
+            try
+            {
+                port.ws_queue_limit =
+                    beast::lexicalCastThrow<std::uint16_t>(result.first);
+
+                // Queue must be greater than 0
```

Comment:
Magic values are bad, but so are disconnects :-) I guess the real aim is to have "trusted" code not get disconnected arbitrarily. Is a better approach to not enforce this limit for local connections or specified ip addresses, much like admin commands can be configured?